ext/standard/tests/file/copy_variation2-win32-mb.phpt

```diff
@@ -22,24 +22,24 @@ fclose($file_handle);
 $dest_files = array(
 
   /* File names containing special(non-alpha numeric) characters */
-  "_copy_variation2.tmp",
-  "@copy_variation2.tmp",
-  "#copy_variation2.tmp",
-  "+copy_variation2.tmp",
-  "?copy_variation2.tmp",
-  ">copy_variation2.tmp",
-  "!copy_variation2.tmp",
-  "&copy_variation2.tmp",
-  "(copy_variation2.tmp",
-  ":copy_variation2.tmp",
-  ";copy_variation2.tmp",
-  "=copy_variation2.tmp",
-  "[copy_variation2.tmp",
-  "^copy_variation2.tmp",
-  "{copy_variation2.tmp",
-  "|copy_variation2.tmp",
-  "~copy_variation2.tmp",
-  "\$copy_variation2.tmp"
+  "_copy_variation2_mb.tmp",
+  "@copy_variation2_mb.tmp",
```

Comment:
Maybe have a variable for `copy_variation2_mb` ?

Also is the `$src_file_name` variable above conflicting or not?

Reply:
I don't think an extra variable is worth it as this test is normally not something that evolves or copy pasted again.

> Also is the $src_file_name variable above conflicting or not?

I don't believe so: there's extra unicode chars at the end. Or am I missing something?